From 8ad28171ac5c3fbd1917a2f52e75423c4d357b24 Mon Sep 17 00:00:00 2001
From: David Brown <nix@davidb.org>
Date: Thu, 3 Jun 2010 06:40:04 -0700
Subject: [PATCH] Fix initializer for 2.7.0 release

The release of 2.7.0 of pidgin/purple gained two extra fields in a
structure.
---
 src/core/sipe.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/core/sipe.c b/src/core/sipe.c
index 45a9015..19f4237 100644
--- a/src/core/sipe.c
+++ b/src/core/sipe.c
@@ -10683,6 +10683,8 @@ PurplePluginProtocolInfo prpl_info =
 	NULL,					/* get_media_caps */
 #if PURPLE_VERSION_CHECK(2,7,0)
 	NULL,					/* get_moods */
+	NULL,					/* set_public_alias */
+	NULL,					/* get_public_alias */
 #endif
 #endif
 #endif
-- 
1.7.1

